src/components/Video/Controls/Controls.module.scss

```diff
@@ -9,13 +9,24 @@
   padding-bottom: 24px;
   font-size: 17px;
 
+  &.isFocusMode {
+    position: fixed;
+    width: 100vw;
```

Comment:
We're mixing a lot of different values in this file. The 100vw is fine, but looking at the use of px values above + rem + the % based sizing in the media query.

Can we settle on one where possible?

Worth n